### ISSUE TITLE
x button kind of looks like multiply, not close, to some people.

### ISSUE THREAD

**Issue by scottkellum** [modularscale/modularscale.com#59]
https://twitter.com/fstorr/status/568566862218289152

**Comment by fstorr** [comment]
This button also needs to be accessible. At the moment, because there's no accessible name (for example, an aria-label) it's impossible for some users of assistive technology to understand what the functionality of that that element is.

**Comment by scottkellum** [comment]
Added titles to SVGs with ARIA attributes pointing to them, but still thinking on design changes to make this more clear.

**Comment by scottkellum** [comment]
I may close without further changes for now. If this area of the page gets redesigned in the future I will take this case to heart.
